AdminWebsite/AdminWebsite/ClientApp/src/app/services/launch-darkly.service.spec.ts

```diff
@@ -33,19 +33,4 @@ describe('LaunchDarklyService', () => {
 
         expect(ldClientSpy.close).toHaveBeenCalled();
     });
-
-    it('should return a given flag', fakeAsync(() => {
```

Comment:
why is this test removed? there's no change in the LD service

Reply:
can we change it to reference a different flag? It would be useful to keep test coverage for this function, to check that it works